### ISSUE TITLE
PasswordState Class: Can't use GetPassword() method after DecryptPassword()

### ISSUE THREAD

**Issue by colombeen** [dnewsholme/PasswordState-Management#14]
You can't use the GetPassword() method after a decrytion has been performed on the property, which should still work in my opinion.

```powershell
PS H:\> $test = Find-PasswordStatePassword -Search test

PS H:\> $test

PasswordID  : 1
Title       : Test
Username    : Test
Password    : EncryptedPassword
Description : 
Domain      : 

PS H:\> $test.GetPassword()
1a2b3c4d5e6f7g8h9i

PS H:\> $test.DecryptPassword()

PS H:\> $test.GetPassword()
Exception calling "SecureStringToBSTR" with "1" argument(s): "Value cannot be null.
Parameter name: s"
At C:\Program Files\WindowsPowerShell\Modules\passwordstate-management\0.0.94\PasswordstateClass.ps1:18 char:9
+         $BSTR = [System.Runtime.InteropServices.Marshal]::SecureStrin ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifiedErrorId : ArgumentNullException
```
We should test if the object is a securestring or not and depending on the result just return the value or decrypt it on the fly.

**Comment by colombeen** [comment]
I already fixed this issue in my fork

**Comment by dnewsholme** [comment]
Agreed i didn't envision this scenario but your solution sounds right. Feel
free to send over a pull request for the fix and I'll merge it in.

**Comment by dnewsholme** [comment]
Fixed in #15 . Thanks for your contribution.

Published to Gallery
https://www.powershellgallery.com/packages/passwordstate-management/0.0.95
